### ISSUE TITLE
[Bug]: getwhisky.app doesn't load in arc browser (quic protocol issue?)

### ISSUE THREAD

**Issue by brooksc** [Whisky-App/Whisky#986]
### Description

unable to load the site in the browser -- and tried via curl below.

### Steps to reproduce

open https://getwhisky.app/ in the browser

Note I tried loading it in arc and it throws an error.  Safari does load it, I don't understand why given the curl response below.  

### Expected behaviour

website loads

### Logs

```bash
curl -v https://getwhisky.app/
*   Trying 18.204.152.241:443...
* Connected to getwhisky.app (18.204.152.241) port 443
* ALPN: curl offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
* Closing connection
curl: (35) LibreSSL/3.3.6: error:1404B42E:SSL routines:ST_CONNECT:tlsv1 alert protocol version
~
❯
curl -v http://getwhisky.app/
*   Trying 18.204.152.241:80...
* Connected to getwhisky.app (18.204.152.241) port 80
> GET / HTTP/1.1
> Host: getwhisky.app
> User-Agent: curl/8.4.0
> Accept: */*
>
< HTTP/1.1 204 No Content
< Server: nginx
< Date: Sat, 18 May 2024 00:20:04 GMT
< Content-Length: 0
< Connection: keep-alive
< Report-To: {"group":"heroku-nel","max_age":3600,"endpoints":[{"url":"https://nel.heroku.com/reports?ts=1715991603&sid=67ff5de4-ad2b-4112-9289-cf96be89efed&s=Lc7FYEpsgp8x1zcuqL15vKdxvjLiSCZKSRwgXKr56SA%3D"}]}
< Reporting-Endpoints: heroku-nel=https://nel.heroku.com/reports?ts=1715991603&sid=67ff5de4-ad2b-4112-9289-cf96be89efed&s=Lc7FYEpsgp8x1zcuqL15vKdxvjLiSCZKSRwgXKr56SA%3D
< Nel: {"report_to":"heroku-nel","max_age":3600,"success_fraction":0.005,"failure_fraction":0.05,"response_headers":["Via"]}
< X-Powered-By: Express
< Etag: W/"a-bAsFyilMr4Ra1hIU5PyoyFRunpI"
< Via: 1.1 vegur
<
* Connection #0 to host getwhisky.app left intact
```

### What version of Whisky are you using?

2.3.2

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by brooksc** [comment]
when trying to load it in arc (which is based on chromium) it shows

This site can’t be reachedThe webpage at https://getwhisky.app/ might be temporarily down or it may have moved permanently to a new web address.
ERR_QUIC_PROTOCOL_ERROR

**Comment by IsaacMarovitz** [comment]
<img width="1392" alt="Screenshot 2024-05-17 at 20 57 41" src="https://github.com/Whisky-App/Whisky/assets/42140194/314037b4-7e3d-4051-8840-027064c05b21">

I cannot recreate this locally. This may be an issue with an extension you have installed.

**Comment by IsaacMarovitz** [comment]
Curl will not work on this site due to Cloudflare's protections.

**Comment by brooksc** [comment]
Thank you for the quick response!

This is bizarre.  I tried loading https://data.getwhisky.app/Wine/WhiskyWineVersion.plist in arc and in safari -- and neither worked.  I was investigating this because I couldn't download WineWhisky and pulled the url from the source.

But it must be my network for some reason!  I tried tethering to my phone and it downloaded just fine.

I'm on Quantum/CenturyLink and perhaps they are blocking (?!?!) or have some bad routing or dns or something.  

Anyway closing this issue.  FYI Issue #943 is probably the same cause - some sort of network/ISP issue.
